Trimble.Modus.Components/Handlers/TMFloatingButtonAndroidTouchHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿#if ANDROID
+using Trimble.Modus.Components;
+using Android.Content;
+using Android.Views;
+using Microsoft.Maui.Controls.Handlers.Compatibility;
+
+namespace Trimble.Modus.Components.Handlers;
+
+public class TMFloatingButtonAndroidTouchHandler : VisualElementRenderer<TMFloatingButton>
```

Comment:
Is this any different from the `TMButtonAndroidTouchHandler`? I don't see any differences. Can you just use the `TMButtonAndroidTouchHandler`?

Reply:
Since secondary has different background color and onClick color separate Handlers are required.